commands/create.go

```diff
@@ -137,7 +137,8 @@ func runCreate(dockerCli command.Cli, in createOptions, args []string) error {
 		if len(args) > 0 {
 			ep, err = validateEndpoint(dockerCli, args[0])
 			if err != nil {
```

Comment:
If this is different things now for different drivers we should just move the validation to the driver level.